.github/workflows/test-integrations.yml

```diff
@@ -324,7 +324,7 @@ jobs:
               ${{ matrix.test-cases }} \
               --target-image ${{ env.CONSUL_LATEST_IMAGE_NAME }} \
               --target-version local \
-              --latest-image ${{ env.CONSUL_LATEST_IMAGE_NAME }} \
+              --latest-image docker.mirror.hashicorp.services/${{ env.CONSUL_LATEST_IMAGE_NAME }} \
```

Comment:
@loshz , this is where we change the docker image to use `docker.mirror.hashicorp.services`. I am not sure where to remove the `docker login` since it seems needed to pull vault image. Please advise.

Reply:
I pushed a change in https://github.com/hashicorp/consul/pull/17186/commits/81e3c44573eae9b5bdd5965e651b4c77709a8084